server_test.go

```diff
@@ -698,7 +699,7 @@ func checkInProgressQueriesAtShutdownServer(t *testing.T, srv *Server, addr stri
 	})
 	defer HandleRemove("example.com.")
 
-	client.Timeout = 10 * time.Second
+	client.Timeout = 100 * time.Millisecond
```

Comment:
I’d switch this to 1s, this timeout should never be hit unless the test fails (unless I’m very wrong). It’s a bit more reliable than lowering it to 100ms.

Reply:
done